0_RootFS/gcc_common.jl

```diff
@@ -332,11 +332,14 @@ function gcc_script(compiler_target::Platform)
             atomic_patch -p1 ${p} || true;
         done
 
+        # Patch bad `movq` argument in glibc 2.17
```

Comment:
Can you please add a comment with a link to where the the patch was found, for future reference?